packages/build/src/npm-packages.ts

```diff
@@ -0,0 +1,65 @@
+import { spawnSync } from 'child_process';
+import path from 'path';
+
+const PLACEHOLDER_VERSION = '0.0.0-dev.0';
+const PROJECT_ROOT = path.resolve(__dirname, '..', '..', '..');
+const LERNA_BIN = path.resolve(PROJECT_ROOT, 'node_modules', '.bin', 'lerna');
+
+export function bumpNpmPackages(version: string): void {
+  if (!version || version === PLACEHOLDER_VERSION) {
+    return;
+  }
+
+  spawnSync(LERNA_BIN, [
+    'version',
+    '--bump', version,
```

Comment:
I think it's just `lerna version <version>` without the `--bump`.

```
lerna version [bump]

Bump version of packages changed since the last release.

Positionals:
  bump  Increment version(s) by explicit version _or_ semver keyword,
        'major', 'minor', 'patch', 'premajor', 'preminor', 'prepatch', or 'prerelease'.     
```